modules/Db2/templates/db2-subscription.template.yaml

```diff
@@ -0,0 +1,24 @@
+###############################################################################
```

Comment:
This should be named filename.yaml.tmpl so that it won't ever be interpreted as a yaml until the necessary variables have been replaced.

Reply:
Fixed